docs/maintain-guides-how-to-monitor-your-node.md

```diff
@@ -0,0 +1,227 @@
+---
+id: maintain-guides-how-to-monitor-your-node
+title: Monitor your node
+sidebar_label: Monitor your node
+---
+
+This guide will walk you through how to set up the [Prometheus](https://prometheus.io/) with [Grafana](https://grafana.com/) to monitor your node using Ubuntu 18.04.
+
+## Preparation
+
+First, create a user for Prometheus and Node Exporter without the possibility to log in.
+
+```bash
+useradd --no-create-home --shell /usr/sbin/nologin prometheus
+useradd --no-create-home --shell /bin/false node_exporter
+```
+
+Create the directories required to store the configuration and executable files.
+
+```bash
+mkdir /etc/prometheus
+mkdir /var/lib/prometheus
+```
+
+Change the ownership of these directories to `prometheus` so that only prometheus can access to these them.
+
+```bash
+sudo chown prometheus:prometheus /etc/prometheus
+sudo chown prometheus:prometheus /var/lib/prometheus
+```
+
+## Installing & Configuring Node Exporter
+
```

Comment:
Perhaps a short paragraph explaining what Node Exporter is, and why we need it if substrate node already exposes prometheus metrics on its own.